pysonnet/projects.py

```diff
@@ -27,10 +34,14 @@ class Project(dict):
     def __init__(self, load_path=None):
         super().__init__()
         self.project_file_path = None
+        self.file_name = file_name
```

Comment:
file_name isn't defined yet so this will error. Also, which file_name is this? Be specific